gwt-material/src/test/java/gwt/material/design/client/ui/MaterialModalTest.java

```diff
@@ -41,21 +41,6 @@ public void init() {
         checkType(modal);
         checkDimissible(modal);
         checkDuration(modal);
-        checkMultipleModalZIndexes();
-    }
-
-    protected void checkMultipleModalZIndexes() {
```

Comment:
I will provide a basic test for this. I will merge this in also so that I can work on tests.